package.json

```diff
@@ -54,18 +54,19 @@
     "pretest": "npm run qunit-fixture && npm run babel:tests && npm run npmcopy",
     "qunit-fixture": "node build/tasks/qunit-fixture.js",
     "start": "node -e \"require('./build/tasks/build.js').buildDefaultFiles({ watch: true })\"",
-    "test:browserless": "npm run pretest && npm run test:node_smoke_tests && npm run test:promises_aplus && npm run test:jsdom",
-    "test:browser": "npm run pretest && npm run build:all && grunt karma:main",
-    "test:esmodules": "npm run pretest && npm run build:main && grunt karma:esmodules",
-    "test:jsdom": "npm run pretest && npm run build:main && grunt test:jsdom",
-    "test:no-deprecated": "npm run pretest && npm run build -- -e deprecated && grunt karma:main",
-    "test:selector-native": "npm run pretest && npm run build -- -e selector && grunt karma:main",
-    "test:slim": "npm run pretest && npm run build -- --slim && grunt karma:main",
+    "test:browser": "npm run pretest && npm run build:main && npm run test:unit -- -b chrome -b firefox --no-isolate -h",
+    "test:browserless": "npm run pretest && npm run build:all && node build/tasks/node_smoke_tests.js && node build/tasks/promises_aplus_tests.js && npm run test:unit -- -b jsdom -m basic",
+    "test:jsdom": "npm run pretest && npm run build:main && npm run test:unit -- -b jsdom -m basic",
     "test:node_smoke_tests": "npm run pretest && npm run build:all && node build/tasks/node_smoke_tests.js",
     "test:promises_aplus": "npm run build:main && node build/tasks/promises_aplus_tests.js",
+    "test:firefox": "npm run pretest && npm run build:main && npm run test:unit -- -v -b firefox -h --no-isolate",
+    "test:safari": "npm run pretest && npm run build:main && npm run test:unit -- -b safari --no-isolate",
     "test:server": "node test/runner/server.js",
+    "test:esm": "npm run pretest && npm run build:main && npm run test:unit -- --esm --no-isolate -h",
+    "test:no-deprecated": "npm run pretest && npm run build -- -e deprecated && npm run test:unit -- --no-isolate -h",
+    "test:selector-native": "npm run pretest && npm run build -- -e selector && npm run test:unit -- --no-isolate -h",
+    "test:slim": "npm run pretest && npm run build -- --slim && npm run test:unit -- --no-isolate -h",
```

Comment:
Can we unify the flags passed to all these? The `-h` parameter is placed in a different place in different scripts, the Safari one doesn't have it at all, etc. The more similar they all are, the better.

Reply:
We can do that, but the Safari one should not have it.

Reply:
Looks like I was consistent except for one. Differences exist in `test:safari` (no `-h`) and `test:firefox`. For the latter, I purposely add `-v` so that we can double confirm that the Firefox version matches ESR (verbose mode logs more browser info when spinning up the driver).